<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tea_Change</class>
 <widget class="QWidget" name="Tea_Change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #efefff,stop:1 #cfcfff);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>输入学生学号和成绩并按确定键</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>学生学号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>110</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>分数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stunumEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="stuscorceEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>109</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>470</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <widget class="QPushButton" name="confimButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>输入教授课程号并按查询显示学生成绩信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>课程号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sectionnumEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>530</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="scoretableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>321</width>
        <height>421</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>学生学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学生名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>成绩</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QPushButton" name="returnButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>530</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>返回教师主界面</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;》》》》教师更改学生成绩界面《《《《&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchsectionButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>440</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查询所有教授的课程号</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="sectiontableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>271</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>课程号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>点击查询所有查看教授的课程信息</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
